recipe/meta.yaml

```diff
@@ -128,7 +128,7 @@ test:
 
 about:
   home: http://fftw.org
-  license: GPL-2.0
+  license: GPL-2.0-only
```

Comment:
I think it should be “or-later”. That’s what is indicated in [this file]( https://github.com/FFTW/fftw3/blob/master/COPYRIGHT ).

> the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.

Reply:
woops! good catch and thanks!

Reply:
Thanks Matt 🙂